pyproject.toml

```diff
@@ -47,19 +47,19 @@ packages = [
 
 
 [tool.poetry.dependencies]
-python = "^3.7.0"
+python = "^3.6.0"
 aiohttp = "^3.5"
 asgiref = "^3.2"
 channels = "^2.2"
-django = "^2.2"
```

Comment:
This would lock versions out of Django 3: https://github.com/datadvance/DjangoChannelsGraphqlWs/issues/35